code/modules/projectiles/guns/smartgun.dm

```diff
@@ -704,5 +710,30 @@
 
 /obj/item/weapon/gun/smartgun/admin
 	requires_power = FALSE
-	requires_powerpack = FALSE
+	requires_battery = FALSE
 	requires_harness = FALSE
+
+/obj/item/smartgun_battery
+	name = "smartgun DV9 battery"
+	desc = "A standard DV9 lithium battery" //yell at lore team for a more soulful description - Morrow
```

Comment:
is this a TODO...

Reply:
A TODO that shouldn't be merged but yes hahahahha